PinpointKit/PinpointKit/Sources/PinpointKit.swift

```diff
@@ -41,7 +41,8 @@ public class PinpointKit {
      - parameter delegate:           A delegate that is notified of significant events.
      */
     public convenience init(feedbackRecipients: [String], delegate: PinpointKitDelegate? = nil) {
```

Comment:
Should this initializer take a `FeedbackConfiguration` now? It seems like it might be common enough to specify a title or body that we should give people those options in a convenience initializer on `PinpointKit`.